cmd/podman/build.go

```diff
@@ -0,0 +1,136 @@
+package main
+
+import (
+	"os"
+	"os/exec"
+
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+var (
+	buildFlags = []cli.Flag{
+		cli.StringSliceFlag{
+			Name:  "build-arg",
+			Usage: "`argument=value` to supply to the builder",
+		},
+		cli.StringSliceFlag{
+			Name:  "file, f",
+			Usage: "`pathname or URL` of a Dockerfile",
+		},
+		cli.StringFlag{
+			Name:  "format",
+			Usage: "`format` of the built image's manifest and metadata",
+		},
+		cli.BoolFlag{
+			Name:  "pull-always",
+			Usage: "pull the image, even if a version is present",
+		},
+		cli.BoolFlag{
+			Name:  "quiet, q",
+			Usage: "refrain from announcing build instructions and image read/write progress",
+		},
+		cli.StringFlag{
+			Name:  "runtime",
+			Usage: "`path` to an alternate runtime",
+		},
+		cli.StringSliceFlag{
+			Name:  "runtime-flag",
+			Usage: "add global flags for the container runtime",
+		},
+		cli.StringFlag{
+			Name:  "signature-policy",
+			Usage: "`pathname` of signature policy file (not usually used)",
+		},
+		cli.StringSliceFlag{
+			Name:  "tag, t",
+			Usage: "`tag` to apply to the built image",
+		},
+		cli.BoolFlag{
+			Name:  "tls-verify",
+			Usage: "require HTTPS and verify certificates when accessing the registry",
+		},
+	}
+	buildDescription = "podman build launches the Buildah command to build an OCI Image. Buildah must be installed for this command to work."
+	buildCommand     = cli.Command{
+		Name:        "build",
+		Aliases:     []string{"build"},
+		Usage:       "Build an image using instructions in a Dockerfile",
+		Description: buildDescription,
+		Flags:       buildFlags,
+		Action:      buildCmd,
+		ArgsUsage:   "CONTEXT-DIRECTORY | URL",
+	}
+)
+
+func buildCmd(c *cli.Context) error {
+
+	budCmdArgs := []string{"bud"}
+
+	for _, buildArg := range c.StringSlice("build-arg") {
+		budCmdArgs = append(budCmdArgs, "--build-arg", buildArg)
+	}
+	for _, fileName := range c.StringSlice("file") {
+		budCmdArgs = append(budCmdArgs, "--file", fileName)
+	}
+	if c.IsSet("format") {
+		budCmdArgs = append(budCmdArgs, "--format", c.String("format"))
+	}
+	if c.IsSet("pull-always") {
+		budCmdArgs = append(budCmdArgs, "--pull-always")
+	}
+	if c.IsSet("quiet") {
+		boolValue := "False"
+		if c.Bool("quiet") {
+			boolValue = "True"
+		}
+		quietParam := "--quiet=" + boolValue
+		budCmdArgs = append(budCmdArgs, quietParam)
```

Comment:
This one can be greatly simplified to just

budCmdArgs = append(budCmdArgs, "--quiet", c.TBool("Quiet"))

Reply:
I can only get it down to one line.  the quiet param is a bit squirlley, it needs to be "quiet=false" or "quiet" or "quiet=true".  No spaces around the = which happens if you do the append as you have it.

I got it down to:

```
                quietParam := "--quiet=" + strconv.FormatBool(c.Bool("quiet"))
                budCmdArgs = append(budCmdArgs, quietParam)
```

and  you're right, more readable.  Thanks for the thought.